src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```diff
@@ -4,23 +4,29 @@
 using System.Linq;
 using System.Windows.Controls;
 using System.Windows.Input;
+using System.Windows.Navigation;
 using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.LanguageServices.Implementation.Utilities;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.Options
 {
     internal partial class GridOptionPreviewControl : AbstractOptionPageControl
     {
+        private const string useEditorConfigUrl = "https://go.microsoft.com/fwlink/?linkid=866541";
```

Comment:
Nit: const variables should be PascalCased.

Reply:
done